CONTRIBUTING.md

```diff
@@ -29,7 +29,7 @@ you will most likely use this for WSL), `homebrew` (macOS / Linux), and more:
 - `git` (package `git` everywhere);
 - A Java 17 or later JDK (packages vary, use Google/DuckDuckGo/etc.).
   - [Adoptium](https://adoptium.net/) has builds for most operating systems.
-  - Paper requires JDK 17 to build, however makes use of Gradle's
+  - Paper requires JDK 17 to build, however, makes use of Gradle's
```

Comment:
Something about this still reads wrong to me. I feel like there needs to be an `it` before `makes use of` to be correct. But please correct me if I'm wrong.

Reply:
It is fine as it is

Reply:
To me, that sounds wrong. `it` would refer to `Paper`, but there is no need to refer to it again, `Paper` is already named, and the connection to `makes use of` is clear.

I double-checked with my [Language checker](https://languagetool.org/?pk_campaign=addon2-popup-logo), and it seems to agree with me.